Sources/BraintreeCard/CreditCardPOSTBody.swift

```diff
@@ -0,0 +1,133 @@
+// swiftlint:disable all
+import Foundation
+
+struct CreditCardPOSTBody: Encodable {
+    var authenticationInsight: Bool?
+    var merchantAccountId: String?
+    var meta: Meta?
+    let creditCard: CreditCard?
+    
+    private var usesGraphQL: Bool
+
+    enum CodingKeys: String, CodingKey {
+        case authenticationInsight
+        case meta = "_meta"
+        case merchantAccountId
+        case creditCard = "credit_card"
+    }
+
+    init(
+        card: BTCard,
+        authenticationInsight: Bool? = nil,
+        merchantAccountId: String?  = nil,
+        meta: Meta? = nil,
+        usesGraphQL: Bool = false
+    ) {
+        self.creditCard = CreditCard(card: card)
+        self.authenticationInsight = authenticationInsight
+        self.merchantAccountId = merchantAccountId
+        self.meta = meta
+        self.usesGraphQL = usesGraphQL
+        
+    }
+
+    struct Meta: Encodable {
+        var integration: String
+        var source: String
+        var sessionId: String
+
+        init(integration: String, source: String, sessionId: String) {
```

Comment:
i think we can just pass `BTClientMetadata` as a parameter

```suggestion
        init(metadata: BTMetadata) {
```